### ISSUE TITLE
Holes in the predicted depth map

### ISSUE THREAD

**Issue by Jayden9912** [nianticlabs/monodepth2#372]
Why dynamic object(with greater or lower speed than camera) in training dataset will cause holes in depth map? Is it because of the pose estimation and the way we calculate appearance loss?

Any discussion is welcomed. Thanks in advance.

**Comment by mrharicot** [comment]
Hi,
We discuss this in the paper Section 3.2.
When following a car, it will _appear static_ in the training triplet as it has the same speed as the car capturing the data. The network tends to associate being static with being infinitely far away, hence "punching a hole".

I hope this helps!
